src/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Build stage
+FROM python:3.12-slim
+
+WORKDIR /app
+
+# Copy the application code into the container
+COPY . .
+
+# Install dependencies and run migrations
+RUN pip install -r requirements.txt && python manage.py migrate
```

Comment:
I would rather not combine two logically different commands into one RUN instruction.